README.md

```diff
@@ -185,62 +227,60 @@ Read more [here](src/Resources/doc/intercept-request-and-response.md).
 ### Symfony Debug Toolbar / Profiler
 <img src="/src/Resources/doc/img/debug_logs.png" alt="Debug Logs" title="Symfony Debug Toolbar - Guzzle Logs" style="width: 360px" />
 
-### Symfony logs
-All requests are logged into symfony default logger (`@logger` service) with next format:
+### Logging
+
+All requests are logged to Symfony's default logger (`@logger` service) with the following (default) format:
 ```
 [{datetime}] eight_points_guzzle.{log_level}: {method} {uri} {code}
 ```
+
 Example:
 ```
 [2017-12-01 00:00:00] eight_points_guzzle.INFO: GET http://api.domain.tld 200
 ```
 
-You can change message format by overriding `eight_points_guzzle.symfony_log_formatter.pattern` parameter. See allowed variables [here][8].
+You can change the message format by overriding the `eight_points_guzzle.symfony_log_formatter.pattern` parameter.
+For all options please refer to [Guzzle's MessageFormatter][8].
 
 ----
 
 ## Suggestions
-Adding aliases:
-If you want to use different names for provided services you can use aliases. This is a good idea if you don't want
-have any dependency to guzzle in your service name.
-``` yaml
-services:
-   crm.client:
-       alias: eight_points_guzzle.client.api_crm
-```
 
-Use Guzzle MockHandler in tests :
-If you want to mock api calls, you can force the clients to use the Guzzle MockHandler instead of the default one.
+### Create aliases for clients
```

Comment:
Is it still useful to create aliases? As for me, it's better to use autowire and do not get client directly from container.

Reply:
Aliases are still useful for manual wiring. But indeed, this section doesn't apply to autowiring. I'll add a note.

Reply:
Done